vllm/engine/metrics.py

```diff
@@ -82,6 +99,12 @@ def __init__(self, labelnames: List[str]):
             documentation="Histogram of end to end request latency in seconds.",
             labelnames=labelnames,
             buckets=[1.0, 2.5, 5.0, 10.0, 15.0, 20.0, 30.0, 40.0, 50.0, 60.0])
+        self.histogram_request_n = Histogram(
+            name="vllm:request_params_n",
+            documentation="Histogram of the n request parameter.",
+            labelnames=labelnames,
+            buckets=[1, 2, 5, 10, 20],
+        )
```

Comment:
If we keep what's currently called `vllm:request_success`, could this be included as another `label` and the histogram could be constructed on the Grafana side?

Reply:
Great suggestion! I'll incorporate the new metrics into the Grafana dashboard demo.

Reply:
This can be resolved now as you've added the Grafana examples